src/DocSystem.jl

```diff
@@ -123,16 +122,14 @@ function docstr(md::Markdown.MD; kws...)
     for (key, value) in kws
         doc.data[key] = value
     end
-    doc
+    return doc
 end
 docstr(other) = other
 
 
 ## Formatting `DocStr`s. ##
 
 
-
-
```

Comment:
Does the formatter remove empty lines? Not opposed to it, just curious where this change came from.

Reply:
It removes empty lines so that there are at max two emtpy lines between code.